src/actions/Listing.js

```diff
@@ -8,11 +8,42 @@ export const ListingConstants = keyMirror(
   {
     FETCH_IDS: null,
     FETCH_IDS_SUCCESS: null,
-    FETCH_IDS_ERROR: null
+    FETCH_IDS_ERROR: null,
+    FETCH_FEATURED_HIDDEN: null
   },
   'LISTING'
 )
 
+export function fetchFeaturedHiddenListings(networkId) {
+  const readListingsFromGist = async (gitstUrl) => {
+    const response = await fetch(gitstUrl)
+    return (await response.text())
+      .split(',')
+      .map(listing => listing.trim())
+  }
+
+  return async function(dispatch) {
+    try{
+      if (process.env.FEATURED_LISTINGS_GIST === undefined || process.env.HIDDEN_LISTINGS_GIST === undefined){
```

Comment:
I believe these env vars are no longer needed not that we pull from git branch.

Reply:
jp true thanks!